app/views/admin/booths/index.html.haml

```diff
@@ -0,0 +1,58 @@
+.row
+  .col-md-12
+    .page-header
+      %h1
+        Booths
+        = "(#{@booths.length})" if @booths.any?
+        .pull-right
+          - if can? :create, Booth
+            = link_to 'Add Booth', new_admin_conference_booth_path(@conference.short_title), class: 'button btn btn-primary'
+      %p.text-muted
+        All the booth requests
+.row
+  .col-md-12
+    .margin-booth-table
+      %table.table.table-striped.table-bordered.table-hover.datatable
+        %thead
+          %th
+            %b ID
+          %th
+            %b Logo
+          %th
+            %b Title
+          %th
+            %b Submitter
+          %th
+            %b Responsibles
+          %th
+            %b State
+          %th
+            %b Actions
+        - @booths.each do |booth|
+          %tr
+            %td
+              = booth.id
+            %td
+              = image_tag(booth.picture.thumb.url, width: '20%')
```

Comment:
When there is no picture we currently see a little square, which makes me wonder if there was a picture, and what happened to it. If there is no picture, we should not show anything.